library/src/skins/default/src/sass/modules/tool/gradebook/_gradebook.scss

```diff
@@ -1738,3 +1738,23 @@ max-height:100px;
 .form-vertical > div.awesomplete {
   display: flex;
 }
+
+.gb-editable,
+.gb-editable:before {
+  height: 34px;
+}
+
+.gb-cell-selected {
+  border: 2px solid #a5dfa7 !important;
+}
+
+.gb-cell-selected:after{
+  content: '';
+  position: absolute;
+  right: -3px;
+  bottom: -3px;
+  width: 6px;
+  height: 6px;
+  background-color: #a5dfa7;
+  border-radius: 999px;
```

Comment:
that's a big border-radius!

Reply:
We don't need that. I will get rid of that. Thanks